l3.py

```diff
@@ -0,0 +1,51 @@
+import requests
+import matplotlib.pyplot as plt
+from matplotlib.animation import FuncAnimation
+
+waluty = ['ZEC', 'LTC', 'DASH']
+url1 = "https://bitbay.net/API/Public/"

```

Comment:
url1, url2? co to mówi?
Proponuję: URL (url1), BASE_CURRENCY (='USD'), REQUEST_TYPE = 'ticker' / 'orderbook' i FORMAT = 'json'